src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -27,14 +27,13 @@ class DateTimeNormalizer implements NormalizerInterface, DenormalizerInterface,
     public const TIMEZONE_KEY = 'datetime_timezone';
 
     private $defaultContext = [
-        self::FORMAT_KEY => \DateTime::RFC3339,
+        self::FORMAT_KEY => \DateTimeInterface::RFC3339,
         self::TIMEZONE_KEY => null,
     ];
 
     private const SUPPORTED_TYPES = [
         \DateTimeInterface::class => true,
         \DateTimeImmutable::class => true,
-        \DateTime::class => true,
```

Comment:
Are you sure to drop support of normalization of `\DateTime`?

Reply:
This part is a no-go. We must still support normalizing `DateTime` objects. Otherwise, we force dropping usages of DateTime in all projects instead (and this is a BC break)